lib/Imagine/Image/Box.php

```diff
@@ -36,8 +36,12 @@
      *
      * @throws InvalidArgumentException
      */
-    public function __construct($width, $height)
+    public function __construct($width, $height = null)
```

Comment:
This behavior is not documented.

Moreover, I'd have used a static shortcut like `Box::square($size)` instead of an optional `height`